atom/browser/common_web_contents_delegate_mac.mm

```diff
@@ -22,17 +23,9 @@
   if (event.windowsKeyCode == ui::VKEY_ESCAPE && is_html_fullscreen())
     ExitFullscreenModeForTab(source);
 
-  BOOL handled = [[NSApp mainMenu] performKeyEquivalent:event.os_event];
-  if (!handled && event.os_event.window) {
-    // Handle the cmd+~ shortcut.
-    if ((event.os_event.modifierFlags & NSCommandKeyMask) /* cmd */ &&
-        (event.os_event.keyCode == 50  /* ~ */)) {
-      if (event.os_event.modifierFlags & NSShiftKeyMask) {
-        [NSApp sendAction:@selector(_cycleWindowsReversed:) to:nil from:nil];
-      } else {
-        [NSApp sendAction:@selector(_cycleWindows:) to:nil from:nil];
-      }
-    }
+  if (event.os_event.window) {
+    AtomNSWindow* native_window = static_cast<AtomNSWindow*>(event.os_event.window);
```

Comment:
> Also I haven't tried, but I think we should be able to just call [window redispatchKeyEvent:] without type conversion.

Yeah, when I try that I get the following compiler error:

`error: 'NSWindow' may not respond to 'redispatchKeyEvent:' [-Werror]`